### DISCUSSION TITLE
show hitboxes, show hitboxes on death and hitbox trial

### DISCUSSION THREAD

**Discussion by mojaksebastian** [Prevter/GDOpenHack#94] [Ideas]
i need it ok its very important i cant breathe without it

**Comment by ghost** [comment]
**I still wait for hitboxes in open hack.,,**

**Comment by dckrell** [comment]
its in the works, meanwhile if you want the latest beta (**CONTAINS HITBOXES**) go to actions, if ur vanilla then go for the "build vanilla mod", if ur on geode go for the "build geode mod", for vanilla you just install it lmao, for geode open the gd folder find the geode folder replace the old mod file (settings carry over) don't blame anyone if it breaks btw

**Comment by Rustring** [comment]
It's added now
